inserted gripper to the park action-designator after put-down

yazdani opened the pull request:
added gripper to the action-designator after putting down the object and fixed some indentation

Commit:
added currently used gripper to action-designator for parking without object

After putting down object the gripper side is not available in the belief state anymore.

Commit:
corrected indentation

Commit:
Merge branch 'master' of https://github.com/yazdani/cram_highlevel

moesenle commented:
Why exactly is this change necessary? 

I would like to get rid of code that is too specific which also includes designator properties such as arm or gripper. If I remember correctly, I had something similar in the past and removed it because the arm to be parked should be inferred in the manipulation process module and unless there is a really good reason for this patch, I'm not too eager to apply it.

If you want, we can discuss the intended design via IM or phone. Just let me know.

Commit:
in park-trajectory remove gripper and add object-desig, also switch properties from `pose parked' to `to park'

Remove the gripper from the action-designator of park-trajectory and add the object-designator to get the used gripper. Also replace the `pose parked' property with the `to park' property and add it to the desig-properties
